site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -55,7 +55,8 @@ export const Language = {
   timezoneLabel: "Timezone",
   ttlLabel: "Time until shutdown (hours)",
   ttlCausesShutdownHelperText: "Your workspace will shut down",
-  ttlCausesShutdownAfterStart: "after its next start",
+  ttlCausesShutdownAfterStart: `after its next start. We delay shutdown by an
+hour whenever we detect activity`,
```

Comment:
Front-end isn't my forte, but wondering why we made this multi-line?

Reply:
^same question as above. Otherwise seems fine.

Reply:
I think in this case it's not ideal to add a literal newline to the string. A better approach may be to `["part 1", "part 2"].join(" ")` (or even `"" + ""`) if it has to be split up.